content/docs/jsx-in-depth.md

```diff
@@ -161,27 +161,27 @@ const components = {
 };
 
 function Story(props) {
-  // Correct! JSX type can be a capitalized variable.
+  // Doğru! JSX türü büyük harfli bir değişken olabilir.
   const SpecificStory = components[props.storyType];
   return <SpecificStory story={props.story} />;
 }
 ```
 
-## Props in JSX {#props-in-jsx}
+## JSX'de Prop'lar {#props-in-jsx}
 
-There are several different ways to specify props in JSX.
+JSX'te prop'ları belirtmenin birkaç farklı yolu vardır.
 
-### JavaScript Expressions as Props {#javascript-expressions-as-props}
+### Prop Olarak JavaScript İfadeleri {#javascript-expressions-as-props}
 
-You can pass any JavaScript expression as a prop, by surrounding it with `{}`. For example, in this JSX:
+Herhangi bir JavaScript ifadesini `{}` ile çevreleyerek bir prop olarak iletebilirsiniz. Örneğin, bu JSX'te:
 
 ```js
 <MyComponent foo={1 + 2 + 3 + 4} />
 ```
 
-For `MyComponent`, the value of `props.foo` will be `10` because the expression `1 + 2 + 3 + 4` gets evaluated.
+`MyComponent` için, `props.foo` değeri `10` olacaktır çünkü `1 + 2 + 3 + 4` ifadesi çalıştırılır.
 
-`if` statements and `for` loops are not expressions in JavaScript, so they can't be used in JSX directly. Instead, you can put these in the surrounding code. For example:
+`if` bildirimleri ve` for` döngüleri JavaScript'te ifade değildir, bu nedenle doğrudan JSX'te kullanılamazlar. Onun yerine, bunları çevreleyen bir koda koyabilirsiniz. Örneğin:
```

Comment:
`bildirimleri -> ifadeleri`

Reply:
or `deyimleri`. but not `bildirimleri`. `bildirim` is more like a declaration.

Reply:
You are absolutely right. But, if I choose `ifadeleri`, the sentence will be like `ifadeleri JavaScript'te ifade değildir`. Which is not making sense. So I'm going with `deyimleri` in this case.